src/Service/Configurations/RuntimeConfigValidator.cs

```diff
@@ -792,84 +691,24 @@ public void ValidateStoredProceduresInConfig(RuntimeConfig runtimeConfig, ISqlMe
                     {
                         throw new DataApiBuilderException(
                             message: e.Message,
-                            statusCode: System.Net.HttpStatusCode.ServiceUnavailable,
+                            statusCode: HttpStatusCode.ServiceUnavailable,
                             subStatusCode: DataApiBuilderException.SubStatusCodes.ConfigValidationError);
                     }
                 }
             }
         }
 
-        /// <summary>
-        /// Pre-processes the permissions section of the runtime config object.
-        /// For eg. removing the @item. directives, checking for invalid characters in claimTypes etc.
-        /// </summary>
-        /// <param name="runtimeConfig">The deserialised config object obtained from the json config supplied.</param>
-        public void ProcessPermissionsInConfig(RuntimeConfig runtimeConfig)
```

Comment:
Are we skipping this now or is this adjusted in the rewrite? I see this method in effect, just removed the `@item.` directives from the policy.

Reply:
Yes, this particular method was no longer needed as the processing happens elsewhere.